codeU1.py

```diff
@@ -0,0 +1,118 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Fri May 19 15:10:45 2017
+
+@author: Luiza
+"""
+
+# -*- coding: utf-8 -*-
+"""
+Created on Thu May 18 10:12:38 2017
+CodeU exercise 1
+@author: Luiza
+"""
+
+# Task 1: Given 2 strings, decide if one of them is a permutation of the other
+# Return:
+#         1: if both strings are empty
+#         0: if one string is empty or they are not a permutation
+#         1: given trimmed strings form a permutation
+
+import numpy as np
+   
+def is_permutation(str1,str2):
+    yes = "'" + str1 + "'" + " is a permutation of " + "'" + str2 + "'";
+    no = "'" + str1 + "'" + " is not a permutation of " + "'" + str2 + "'";
+    str1 = str1.strip().lower()
+    str2 = str2.strip().lower()
+    if  not str1 and not str2:
+        print "Two empty strings are permutations"

```

Comment:
Also maybe take a look at python logging library:

https://docs.python.org/3.6/howto/logging.html#logging-basic-tutorial

It is not at all important here, but in general one would use logging to output diagnostic information rather than direct printing.